Source/Core/Core/PowerPC/Jit64/Jit_LoadStore.cpp

```diff
@@ -315,7 +315,10 @@ void Jit64::dcbz(UGeckoInstruction inst)
 	MOV(32, R(RSCRATCH), gpr.R(b));
 	if (a)
 		ADD(32, R(RSCRATCH), gpr.R(a));
-	AND(32, R(RSCRATCH), Imm32(~31));
+	if (SConfig::GetInstance().m_LocalCoreStartupParameter.bWii)
+		AND(32, R(RSCRATCH), Imm32(~31));
+	else
+		AND(32, R(RSCRATCH), Imm32(Memory::RAM_MASK & ~31));
```

Comment:
1882 fixes the crash, but only if MMU is disabled...?  Odd; if you're not getting any PanicAlerts, the only possible causes for behavior change are memcheck code generation differences and the presence of the FakeVMem region, neither of which should have any effect; I'd appreciate it if you'd help me track down whatever the issue is.

In any case, what are the DBATs set to?  If the answer is "this software requires dynamic-bat", then clearly we shouldn't be making any changes here.